access/legacy/handler.go

```diff
@@ -15,12 +15,14 @@ import (
 
 type Handler struct {
 	api   access.API
+	psapi access.PROTOCOL_STATE_API
```

Comment:
You dont' need to change the legacy handler! this is only for the new "observer"! https://github.com/onflow/flow-go/pull/2370/files#diff-c396d8ee377761a43fca284545cb4ee535f868ea2b16db01db81a21741855ed8R1-R28

Reply:
Sure will make the change